CHANGES.md

```diff
@@ -1,6 +1,473 @@
 Release Changes List
 ====================
 
+version 1.17.0
+==============
+
+Twentieth public release of Chapel, April 5, 2018
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* tool / configuration highlights:
+  - added support for bash autocompletion for `chpl` compiler options
+  - `chpl` now offers suggestions for unrecognized flags
+  - changed the generated executable's name from the main module to its filename
+  - improved support for `chpl`s LLVM back-end
+  - added support for multiple mason repositories, including local registries
+* language highlights:
+  - dramatically improved support for user-defined and implicit initializers
+  - added support for uninterpreted triple-quoted strings
+  - notable improvements to `Owned(c)` and `Shared(c)`
+* standard library/package highlights:
+  - `LinearAlgebra` matrices now use 1-based indices by default
+  - new Kronecker, Cholesky, eigenvector/value routines in `LinearAlgebra`
+  - added the Blowfish cipher to the `Crypto` module
+  - added support for joinPath(), isAbsPath(), commonPath to the `Path` module
+  - added Bessel fucntions to `Math` module
+  - added new channel.advancePastByte(), mark(), commit(), and revert() methods
+  - increased the use of error-handling in standard, package, internal modules
+* performance / benchmark / memory highlights::
+  - significant performance improvements when executing using Cray networks
+  - added a new faster version of the reverse-complement benchmark
+  - closed a number of minor memory leaks
+* portability highlights:
+  - ported Chapel to the Cray XC50 system with ARM processors
+  - improved portability to FreeBSD and PowerPC
+  - improved portability for specific versions of gcc
+* documentation highlights:
+  - moved Chapel's website from http://chapel.cray.com to https://chapel-lang.org
+  - added a color-coded version selection menu to the online documentation
+
+File / Directory Structure
+--------------------------
+* made `make install` install mason
+* added the CHPL_COMM_SUBSTRATE setting to GASNet-specific paths
+
+New Tools / Tool Changes
+------------------------
+* added support for bash autocompletion for `chpl` compiler options
+  (see https://chapel-lang.org/docs/1.17/usingchapel/compiling.html#tab-autocompletion-for-flags)
+* changed the generated executable's name from the main module to its filename
+  (see the `-o` flag on https://chapel-lang.org/docs/1.17/usingchapel/man.html)
+* added support for multiple mason repositories, including local registries
+  (see https://chapel-lang.org/docs/1.17/tools/mason/mason.html#local-registries)
+* added support for `mason help` and `mason version` commands
+* `chpl` now offers suggestions for unfamiliar flags
+  (see `chpl -fast`)
```

Comment:
I find 'for unfamiliar flags' a bit weird, since it might mean unfamiliar to the user. (Can the compiler be familiar with anything?). I had 'unhandled'. How about 'unrecognized' ?

I'd prefer it said
```
 (see e.g. error message for `chpl -fast`)
```

Reply:
Yeah, I agree and used "unrecognized" in the "featured" list, but forgot to change this one too.